CMakeLists.txt

```diff
@@ -2,6 +2,15 @@ cmake_minimum_required(VERSION 3.10)
 project(JasmineGraph)
 set(CMAKE_CXX_STANDARD 17)
 
+# Set build type for VTune profiling if not specified
+if(NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE "RelWithDebInfo")
+    message(STATUS "Build type not specified, setting to RelWithDebInfo for VTune profiling")
+endif()
+
+# Always enable debug symbols for VTune, regardless of build type
+set(CMAKE_ENABLE_DEBUG ON)
```

Comment:
Can we enable this only for debugging purposes? When we build optimized code, having debug symbols enables will make the code to execute less optimal manner.

Reply:
This has been addressed